### PR TITLE
Updating "unified audit logs" to Microsoft 365 Audit

### PR DESCRIPTION
**Pull request by sujitnaray**
Microsoft 365 Audit is the product name.

### CODE REVIEW

**Comment by sujitnaray** [comment]
@robmazz , @markjjo - Could you please approve the changes to reflect "Microsoft 365 Audit" as the product name? Thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/communication-compliance-siem.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ search.appverid:
...
+A common need for organizations is to integrate communication compliance alerts and these SIEM solutions. With this integration, organizations can view communication compliance alerts in their SIEM solution and then remediate alerts within the communication compliance workflow and user experience. For example, an employee sends an offensive message to another employee and that message is detected by a communication compliance policy monitoring for offensive language. These events are tracked in the Microsoft 365 Audit (also known as "Unified Audit Log") by the communication compliance solution and imported in the SIEM solution. An alert is then triggered in the SIEM solution for the organization from events monitored in the Microsoft 365 Audit that are associated with communication compliance alerts. Investigators are notified of the alert in the SIEM solutions and then they investigate and remediate alert in the communication compliance solution.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/communication-compliance-siem.md]:
```suggestion
A common need for organizations is to integrate communication compliance alerts and these SIEM solutions. With this integration, organizations can view communication compliance alerts in their SIEM solution and then remediate alerts within the communication compliance workflow and user experience. For example, an employee sends an offensive message to another employee and that message is detected by a communication compliance policy monitoring for offensive language. These events are tracked in the Microsoft 365 Audit (also known as "unified audit log") by the communication compliance solution and imported into the SIEM solution. An alert is then triggered in the SIEM solution for the organization from events monitored in the Microsoft 365 Audit that is associated with communication compliance alerts. Investigators are notified of the alert in the SIEM solutions and then they investigate and remediate the alert in the communication compliance solution.
```

File: microsoft-365/compliance/communication-compliance-siem.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ IsValid: True
 ObjectState: Unchanged
 ```
 
-**Example of an audit log entry for a policy with custom keyword match (custom sensitive information type):**
+**Example of an Audit log entry for a policy with custom keyword match (custom sensitive information type):**
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/communication-compliance-siem.md]:
```suggestion
**Example of a Microsoft 365 Audit log entry for a policy with custom keyword match (custom sensitive information type):**
```

File: microsoft-365/compliance/communication-compliance-siem.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ ObjectState: Unchanged
 ```
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/communication-compliance-siem.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/compliance/communication-compliance-siem.md [hunk 4]
```diff
@@ -63,11 +63,11 @@ ObjectState: Unchanged
...
->Currently, there may be up to a 24h delay between the time a policy match is recorded in the Microsoft 365 unified audit log and the time in which you can investigate policy matches in communication compliance.
+>Currently, there may be up to a 24h delay between the time a policy match is recorded in the Microsoft 365 Audit and the time in which you can investigate policy matches in communication compliance.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/communication-compliance-siem.md]:
```suggestion
> Currently, there may be up to a 24-hour delay between the time a policy match is recorded in the Microsoft 365 Audit and the time in which you can investigate policy matches in communication compliance.
```

File: microsoft-365/compliance/communication-compliance-siem.md [hunk 5]
```diff
@@ -83,7 +83,7 @@ When you're using the Azure Sentinel to aggregate communication compliance polic
     | where OfficeWorkload == "Exchange" and Operation == "SupervisionRuleMatch" and UserId == "User1@Contoso.com"
     | sort by TimeGenerated*
 
-For more information about the audit logs for Office 365 collected by Azure Sentinel, see [Azure Monitor Logs reference](/azure/azure-monitor/reference/tables/OfficeActivity).
+For more information about the Audit logs for Office 365 collected by Azure Sentinel, see [Azure Monitor Logs reference](/azure/azure-monitor/reference/tables/OfficeActivity).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/communication-compliance-siem.md]:
```suggestion
For more information about the Microsoft 365 Audit logs for Office 365 collected by Azure Sentinel, see [Azure Monitor Logs reference](/azure/azure-monitor/reference/tables/OfficeActivity).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sujitnaray, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by sujitnaray — COMMENTED** [review]
Approved Johan's suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by sujitnaray** [comment]
@JohanFreelancer9 , @robmazz  - I think I have approved all your changes.  Please submit! Or let me know if you need anything else from me.

**Comment by JohanFreelancer9** [comment]
@sujitnaray - thank you for committing the changes. Nothing more is needed from you at this time. The PR will be merged into live once it has been approved by the doc owner/manager.

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@sujitnaray  and @yogkumgit - Is 'the Microsoft 365 Audit" really correct reference throughout this change? Sounds grammatically incorrect. I'll also reach out to the audit content writer about this, I assume all that content would need to match/sync.

**Comment by sujitnaray** [comment]
@robmazz - Yes, "Microsoft 365 Audit" is the product name, and not "unified audit logs".  Adding @markjjo who is the Audit content writer to look over the changes too.

**Comment by robmazz** [comment]
@sujitnaray - thanks. I also spoke with Mark so we could be consistent in the docs. This PR still has some grammar issues, for example - instead of saying 'the Microsoft 365 Audit', we should just say 'Microsoft 365 Audit'. But I'll accept and push this PR and change those items in a separate follow-on PR later today.

**Pull request merged**